src/com/griddynamics/main/Triangle.java

```diff
@@ -0,0 +1,69 @@
+package com.griddynamics.main;
+
+import java.util.Scanner;
+
+public class Triangle {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Please, enter the first side of triangle: ");
+        double first = in.nextDouble();
+        System.out.print("Enter the second side of triangle: ");
+        double second = in.nextDouble();
+        System.out.print("Enter the third side of triangle: ");
+        double third = in.nextDouble();
+        triangleAnalysisSystem(first, second, third);
+
+    }
+
+    public static boolean doubleEquals(double a, double b) {
```

Comment:
@kolegova-mary I would suggest not to use double in that case, but to use BigDecimal instead of - https://javarush.ru/groups/posts/2274-kak-ispoljhzovatjh-bigdecimal-v-java